service/switchboardapp/pom.xml

```diff
@@ -58,7 +71,10 @@
 				</exclusion>
 			</exclusions>
 		</dependency>
-
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-data-jpa</artifactId>
+        </dependency>
```

Comment:
On my local its fine but on GitHub you are right, whats causing that, do you have any idea?

Reply:
I'm don't have any ideas of the top of my head, but maybe we could avoid issues like this by enforcing a CodeStyle file in IntelliJ for all the team. What do you think about it?

Reply:
Thats a good idea, but i just found out we can install 'save actions' plugin and it will reformat the file, organize imports, etc.. on saving a file(intellij autosaves, might not be ideal to reformat in the middle of coding) or on pressing ctrl+shift+s.

Reply:
it fixed the issue! codestyle to enforce the styles is still good suggestion to enforce the styles